### PR TITLE
Add HealthKit types

### PR DESCRIPTION
**Pull request by alemannosara**
This PR adds data types relative to HealthKit records.

### CODE REVIEW

File: dist/event-types.json [hunk 1]
```diff
@@ -34,7 +34,199 @@
...
+                "dateInterval",
+                "type"
+              ]
+            }
+          },
+          "totalFlightsClimbed": {
+            "description": "The total number of flights of stairs climbed during the workout. Unit: count",
+            "type": "number"
+          },
+          "totalSwimmingStokeCount": {
```
Comment (princesseana) [inline dist/event-types.json:221]:
```
"totalSwimmingStrokeCount":
```

File: dist/event-types.json [hunk 2]
```diff
@@ -517,6 +879,16 @@
         }
       }
     },
+    "gas-consumption": {
+      "description": "The quantity of gas consumed.",
+      "formats": {
+        "mlpkgmin": {
+          "description": "Mili liters per kilogram and minute",
```
Comment (princesseana) [inline dist/event-types.json:886]: Milliliters per kilogram and minute

File: dist/event-types.json [hunk 3]
```diff
@@ -2344,6 +2833,16 @@
         }
       }
     },
+    "gas-consumption": {
+      "formats": {
+        "mlpkgmin": {
+          "name": {
+            "en": "Mili liters per kilogram and minute",
+            "fr": "Mili litres par kilogramme et minute"
```
Comment (princesseana) [inline dist/event-types.json:2841]: Milliliters

File: dist/flat.json [hunk 4]
```diff
@@ -367,6 +555,11 @@
       "description": "Microhertz",
       "type": "number"
     },
+    "gas-consumption/mlpkgmin": {
+      "description": "Mili liters per kilogram and minute",
```
Comment (princesseana) [inline dist/flat.json:559]: Milliliters

File: dist/flat.json [hunk 5]
```diff
@@ -1924,6 +2219,12 @@
       },
       "symbol": "rpm"
     },
+    "gas-consumption/mlpkgmin": {
+      "name": {
+        "en": "Mili liters per kilogram and minute",
+        "fr": "Mili litres par kilogramme et minute"
```
Comment (princesseana) [inline dist/flat.json:2225]: Milliliters
Millilitres

File: dist/flat.json [hunk 6]
```diff
@@ -3246,7 +3554,161 @@
...
+                "dateInterval",
+                "type"
+              ]
+            }
+          },
+          "totalFlightsClimbed": {
+            "description": "The total number of flights of stairs climbed during the workout. Unit: count",
+            "type": "number"
+          },
+          "totalSwimmingStokeCount": {
```
Comment (princesseana) [inline dist/flat.json:3702]:
````
"totalSwimmingStrokeCount":
```

File: dist/flat.min.json [hunk 7]
```diff
@@ -367,6 +555,11 @@
       "description": "Microhertz",
       "type": "number"
     },
+    "gas-consumption/mlpkgmin": {
+      "description": "Mili liters per kilogram and minute",
```
Comment (princesseana) [inline dist/flat.min.json:559]: Milliliters

File: src-classes/gas-consumption.json [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    "gas-consumption": {
+      "description": "The quantity of gas consumed.",
+      "formats": {
+        "mlpkgmin": {
+          "description": "Mili liters per kilogram and minute",
+          "type": "number",
+          "extras": {
+            "name": {
+              "en": "Mili liters per kilogram and minute",
+              "fr": "Mili litres par kilogramme et minute"
```
Comment (princesseana) [inline src-classes/gas-consumption.json:11]: Milliliters
Millilitres

File: src-classes/gas-consumption.json [hunk 9]
```diff
@@ -0,0 +1,18 @@
+{
+    "gas-consumption": {
+      "description": "The quantity of gas consumed.",
+      "formats": {
+        "mlpkgmin": {
+          "description": "Mili liters per kilogram and minute",
```
Comment (princesseana) [inline src-classes/gas-consumption.json:6]: Milliliters

File: src-classes/activity.json [hunk 10]
```diff
@@ -5,7 +5,127 @@
...
+              "dateInterval",
+              "type"
+            ]
+          }
+        },
+        "totalFlightsClimbed": {
+          "description": "The total number of flights of stairs climbed during the workout. Unit: count",
+          "type": "number"
+        },
+        "totalSwimmingStokeCount": {
```
Comment (princesseana) [inline src-classes/activity.json:120]:
````
"totalSwimmingStokeCount":
```

**Comment by alemannosara** [comment]
Thanks @princesseana for the review ! I have just committed the changes, as suggested, for milliliters and strokes (for swimming). Please, let me know if there is anything else I can do.

**Pull request merged**